packages/starlight/translations/pl.json

```diff
@@ -0,0 +1,30 @@
+{
+  "skipLink.label": "Przejdź do głównej zawartości",
+  "search.label": "Szukaj",
+  "search.shortcutLabel": "(Naciśnij /, aby wyszukać)",
+  "search.cancelLabel": "Anuluj",
+  "search.devWarning": "Wyszukiwanie jest dostępne tylko w buildach produkcyjnych. Spróbuj zbudować i uruchomić aplikację, aby przetestować lokalnie.",
+  "themeSelect.accessibleLabel": "Wybierz motyw",
+  "themeSelect.dark": "Ciemny",
+  "themeSelect.light": "Jasny",
+  "themeSelect.auto": "Auto",
+  "languageSelect.accessibleLabel": "Wybierz język",
+  "menuButton.accessibleLabel": "Menu",
```

Comment:
Same question, only to confirm this is expected.

Reply:
Correct, a borrowing from French.